src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -189,258 +176,303 @@ const FixPhoneNumbers: React.FC<Props> = ({
+                                {t('DataServer')}
+                              </MenuItem>
+                            </Select>
+                            <Button
+                              variant="contained"
+                              onClick={() => handleBulkConfirm(values)}
+                              data-testid="source-button"
+                            >
+                              <Icon
+                                path={mdiCheckboxMarkedCircle}
+                                size={0.8}
+                                className={classes.buttonIcon}
+                              />
+                              {t('Confirm {{amount}} as {{source}}', {
+                                amount: data.people.totalCount,
+                                source: defaultSource,
+                              })}
+                            </Button>
+                          </Box>
+                        </Box>
+                      </Grid>
+
+                      <Grid item xs={12}>
+                        {values.people.map(
+                          (person: PersonInvalidNumberFragment, i: number) => (
+                            <Contact
+                              key={person.id}
+                              person={person}
+                              personIndex={i}
+                              handleDelete={handleDeleteModalOpen}
+                              setContactFocus={setContactFocus}
+                              handleUpdate={updatePhoneNumber}
+                              errors={errors}
+                              values={values}
+                              setValues={setValues}
+                            />
+                          ),
+                        )}
+                      </Grid>
+
+                      <Grid item xs={12}>
+                        <Box className={classes.footer}>
+                          <Typography>
+                            <Trans
+                              defaults="Showing <bold>{{value}}</bold> of <bold>{{value}}</bold>"
+                              shouldUnescape
+                              values={{ value: data.people.totalCount }}
+                              components={{ bold: <strong /> }}
+                            />
+                          </Typography>
+                        </Box>
+                      </Grid>
+                    </Grid>
+                    <DeleteModal
+                      modalState={deleteModalState}
+                      handleClose={handleDeleteModalClose}
+                      handleDelete={() => handleDelete(values, setValues)}
                     />
-                  </Typography>
-                </Box>
-              </Grid>
-            </>
+                  </>
+                );
+              }}
+            </Formik>
           ) : (
             <NoData tool="fixPhoneNumbers" />
           )}
-        </Grid>
+          ;
```

Comment:
This semicolon can be removed

Reply:
I think it has to do with the fact that you are using the function syntax of `(params): retType => { return x; }` instead of `(params): retType => x`